### DISCUSSION TITLE
Welcome to slackdump Discussions!

### DISCUSSION THREAD

**Discussion by rusq** [rusq/slackdump#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mayookhl** [comment]
Fantastic effort, very stable in the way it runs, robust (no crashes etc). In terms of features, is there a way to run this for specific dates? Eg just one day? (Unfortunately, I am only a user, so I can help with testing it and suggesting features)

**Reply by rusq** [reply]
Thank you for the feedback :) there's no way to do that currently, but that's a great suggestion, shouldn't be hard to implement (famous last words)

**Reply by rusq** [reply]
@mayookhl it's in the latest release, you can use `-dump-from` and `-dump-to` params, the date format is the following '2006-01-31T15:04:05'. It works both on channels and on threads.

**Comment by mootari** [comment]
@rusq Just wanted to join in to say thank you so much for creating slackdump! It's a joy to use and shows the care that went into it.

**Reply by rusq** [reply]
Haha, thank you for the feedback @mootari , glad to hear!

**Reply by rusq** [reply]
Thank you for your support, Fabian :)

**Reply by mootari** [reply]
You're very welcome! I hope that will cover all those coffees! (or teas, if you prefer that) 😁

**Reply by rusq** [reply]
It sure will!

**Comment by samhocevar** [comment]
Allow me to add to the praise. This tool is extremely robust and well written. Excellent work!

A few things in the offline viewer that are probably a matter of taste and not worth a PR or an issue:
 - the channel sidebar is a bit too narrow for most channel names; I set it to `24rem` instead of `16rem`
 - when using the dark theme, links are hard to read because they keep the default dark blue color (at least in Chrome)
 - the _(archived)_ suffix in archived channels is taking a lot of visual space; I changed it to a 🗃️ icon and made it a prefix

Also a question: is support for emoji reactions planned or desirable?
